test/test_port.cpp

```diff
@@ -0,0 +1,63 @@
+/*********************************************************************
+ *        _       _         _
+ *  _ __ | |_  _ | |  __ _ | |__   ___
+ * | '__|| __|(_)| | / _` || '_ \ / __|
+ * | |   | |_  _ | || (_| || |_) |\__ \
+ * |_|    \__|(_)|_| \__,_||_.__/ |___/
+ *
+ * www.rt-labs.com
+ * Copyright 2020 rt-labs AB, Sweden.
+ *
+ * This software is dual-licensed under GPLv3 and a commercial
+ * license. See the file LICENSE.md distributed with this software for
+ * full license information.
+ ********************************************************************/
+
+#include "utils_for_testing.h"
+#include "mocks.h"
+
+#include "pf_includes.h"
+
+#include <gtest/gtest.h>
+
+class PortUnitTest : public PnetUnitTest
+{
+};
+
+TEST_F (PortUnitTest, PortCheckItetator)
+{
+   pnet_t netstruct; /* Do not care about actual value */
+   pf_port_iterator_t port_iterator;
+   int port;
+   int ix;
+
+   /* Check precondiiton. Ports must be enabled for the stack to be working. */
+   EXPECT_TRUE (PNET_MAX_PORT > 0);
```

Comment:
Remove this. That PNET_MAX_PORT is at least 1 should be enforced at compile-time.

Reply:
Done